js/models/TimeTrigger.js

```diff
@@ -4,8 +4,8 @@ TimeTrigger.schema = {
   primaryKey: 'id',
   properties: {
   	id: 'string',
-  	formId: 'string',
-  	surveyId: 'string',
+  	formId:  {type: 'string', default: 'none'},
+  	surveyId:  {type: 'string', default: 'none'},
```

Comment:
The reason I don't like defaulting to a 'none' string, is that its a truthy value and could lead to a potential bug.